ddtrace/contrib/internal/openai/utils.py

```diff
@@ -300,11 +318,12 @@ def _set_token_metrics(span, integration, response, prompts, messages, kwargs):
         usage = response[0].get("usage", {})
         prompt_tokens = getattr(usage, "prompt_tokens", 0)
         completion_tokens = getattr(usage, "completion_tokens", 0)
+        total_tokens = getattr(usage, "total_tokens", 0)
```

Comment:
Why did we choose to add this here? And I think there is a line below that resets
`total_tokens = prompt_tokens + completion_tokens`

Reply:
We can actually get the total tokens from the returned TokenUsage chunk in the response instead of having to calculate. We only do the actual calculation if we have to do the prompt/completion token computation ourselves.